ui/nuxeo-video/nuxeo-video-info.js

```diff
@@ -30,27 +31,24 @@ import '@polymer/iron-flex-layout/iron-flex-layout.js';
     static get template() {
       return html`
         <style>
-          .properties {
-            column-width: 25em;
-            column-count: 3;
+          .damProperties label {
+            min-width: 10em;
+            margin-inline-end: 12px;
           }
 
-          .properties label {
-            width: 10em;
-            overflow: hidden;
-            text-overflow: ellipsis;
-          }
-
-          .item {
+          .damProperties .item {
             @apply --layout-horizontal;
             @apply --layout-flex;
+            line-height: 2.2rem;
+            padding: 0 5px;
+            justify-content: space-between;
+            margin-left: -5px;
           }
-
-          .item label {
-            color: var(--secondary-text-color, #939caa);
+          .damProperties .item:hover {
+            @apply --nuxeo-block-hover;
           }
         </style>
-        <div class="properties">
+        <div class="damProperties">
```

Comment:
Thinking further about this name, since `dam` addon/package doesn't exist anymore, it may not be the most consensual name. Maybe keeping `properties` would be a good option.

Reply:
I agree. Besides, and for future reference, I couldn't find any css classname in camelcase in our codebase.

Reply:
cool, changed it back to properties

Reply:
Maybe this one was forgotten?

Reply:
sorry :( now it should be good.